File: windows-client/PolyPaint/Modeles/Image.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace PolyPaint.Modeles
{
    public class Image
    {
        public string id { get; set; }
        public string ownerId { get; set; }
        public string title { get; set; }
        public string protectionLevel { get; set; }
        public string password { get; set; }
        public string thumbnailUrl { get; set; }
        public string fullImageUrl { get; set; }
    }
}

File: poly-paint-ios/poly-paint-ios/Image.swift
//
//  Image.swift
//  poly-paint-ios
//
//  Created by JP Cech on 10/17/18.
//  Copyright © 2018 PolyAcme. All rights reserved.
//

import UIKit

class Image: NSObject {
    
    var id: String?
    var ownerId: String?
    var title: String?
    var protectionLevel: String?
    var password: String?
    var thumbnailUrl: String?
    var fullImageUrl: String?
    var thumbnail: UIImage?
    var fullImage: UIImage?
    
    func getThumbnailUrl()  -> URL? {
        if let url = URL(string :thumbnailUrl!){
            return url
        }
        return nil
    }
    
    func getFullImageUrl()  -> URL? {
        if let url = URL(string :fullImageUrl!){
            return url
        }
        return nil
    }
    
}
